nixnet/database/_frame.py

```diff
@@ -553,27 +556,21 @@ def mux_subframes(self):
         return self._mux_subframes
 
     @property
-    def pdus(self):
-        # actually returns typing.Iterable[_pdu.Pdu], but avoiding a circular import
-        # type: () -> typing.Iterable[typing.Any]
+    def pdu_properties(self):
+        # type: () -> typing.Iterable[types.PduProperties]
         """list of :any:`Pdu`: Get or set a list that maps existing PDUs to a frame.
 
         A mapped PDU is transmitted inside the frame payload when the frame is transmitted.
         You can map one or more PDUs to a frame and one PDU to multiple frames.
 
         Mapping PDUs to a frame requires setting three frame properties.
-        All three properties are lists of values:
+        All three properties are combined into a PduProperties tuple.
 
-        *   :any:`Frame.pdus`: Set this property first to define
-            the sequence of values for the other two properties.
-        *   :any:`Frame.pdu_start_bits`: Defines the start bit of the PDU inside the frame.
-        *   :any:`Frame.pdu_update_bits`: Defines the update bit for the PDU inside the frame.
+        *   :any:`nixnet.types.PduProperties.ref`: Defines the sequence of values for the other two properties.
```

Comment:
For consistency with other db docs
```
:any:`PduProperties.pdu`
:any:`PduProperties.start_bit`
:any:`PduProperties.update_bit`
```

Rename ref to pdu, per my comment later in the review.